irc.py

```diff
@@ -438,11 +439,25 @@ def main() -> None:
                 'authentication, it\'s recommended to only allow local connections\n' \
                 'you can override this with -o')
 
-    sl_client = slack.Slack(args.tokenfile)
+    if "PORT" in environ:
+        port = environ["PORT"]
```

Comment:
can you cast to int directly here? At the moment port is `Union[str,int]`

Reply:
I cast to int when calling serversocket.bind() because I found that it was failing on my system when port was cast here but not in the args.port case. So it'll either be: 

port = int(environ["PORT"])
port = int(args.port)
serversocket.bind((args.ip, port))
or
port = environ["PORT"]
port = args.port
serversocket.bind((args.ip, int(port)))

I guess I could also do:
port = environ["PORT"]
port = args.port
port=int(port)
serversocket.bind((args.ip, port))

Any of those are fine by me, I went with the one that was the most direct.

Reply:
Now that I think about it, that might be because of the dropped type hint in slack.py.  Looks like re-adding the type hint fixes the args.port part so I'm moving the explicit cast.